index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alan x Walkers - Unity- YouTube</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link rel="stylesheet" href="index.css" />
+        
+</head>
+<body>
+    <nav>
+        <div>
+            <i class="material-icons dp48">dehaze</i>
+            <img class="brand" src="./images/YouTube-Logo.wine.svg"  alt="YouTube Logo"/>
+        </div>
+        <div class="nav-form">
+            <form class="inline-form" action="#">
+                <div class="inline-form-group-1"><input type="search" name="search"  placeholder="Search" id="search"></div>
+                <div class="inline-form-group-2"><button type="submit" class="material-icons">search</button></div>
+            </form>
+        </div>
+        <div>
+            <i class="material-icons dp48">video_call</i>
+            <i class="material-icons dp48">apps</i>
+            <i class="material-icons dp48">notifications</i>
+            <img src="./images/img_avatar3.png" class="avatar" alt="User Profile">
```

Comment:
- [ ] The user profile image is also not present on your clone. Please see my comment above for more info on how to fix this. 

| Source Page	| Clone Page |
|:---:|:---:|
|![Screen Shot 2020-07-13 at 7 54 20 PM](https://user-images.githubusercontent.com/48804258/87336756-a67dd800-c542-11ea-9d18-32d4e1245696.png) |![Screen Shot 2020-07-13 at 7 54 40 PM](https://user-images.githubusercontent.com/48804258/87336789-b1d10380-c542-11ea-8265-32b6b1dfa6aa.png)
|